package.json

```diff
@@ -122,7 +122,7 @@
     "styled-components": "^5.0.1",
     "tiny-invariant": "^1.1.0",
     "tiny-warning": "^1.0.3",
-    "yarn-deduplicate": "^2.1.1"
+    "yarn-deduplicate": "^3.0.0"
```

Comment:
This isn't mentioned in the PR/commit description and doesn't seem to be related. 

The breaking changes don't seem to [affect our usage](https://github.com/atlassian/yarn-deduplicate/blob/master/CHANGELOG.md#300---2020-10-29) and it doesn't seem to have any new features or bugfixes, so I don't think there's much need to upgrade this.

If you still want to make this change, I would say please make a separate PR for that.

Reply:
Sorry, I updated this trying to fix some errors but it turned out I did not understand how yarn-deduplicate worked. I will update the pull request tomorrow as it is time to go to bed where I live.

Reply:
Ok, no rush since I'm gonna have to hold on merging this anyway as it's breaking per our conversation above.

Reply:
Looks like this was removed in the latest update to the PR